File: Algorithms/fibonacci.c
#include <stdio.h>
void print_fibonacci(int num)
{
    int a = 0 , b = 1;
    printf("%d\n%d",a,b);
    num = num - 2;
    while(num > 0)
    {
        num = num - 1;
        sum = a + b;
        a = b;
        b = sum;
        printf("%d\n",sum);

    }
}

void main(){
     clrscr();
     int n;
     printf("Enter the number of terms of Fibonacci series to be printed");
     scanf("%d",&n);

     
}

File: Algorithms/Fibonacci.java
import java.util.*;

public class Fibonacci {
    
public static void main()
{
    
     int n;
     Scanner sin =  new Scanner(System.in);
     System.out.println("Enter the number of terms of Fibonacci series to be printed");
     n = sin.nextInt();
     print_fibonacci(n)


     
}
void print_fibonacci(int num)
{
    int a = 0 , b = 1;
    System.out.println(a + "\n " + b );
    num = num - 2;
    while(num > 0)
    {
        num = num - 1;
        sum = a + b;
        a = b;
        b = sum;
        System.out.println("\n "+ sum);

    }
}

}